swan-lake/integration-technology-tools/edi-tool.md

```diff
@@ -6,4 +6,349 @@ keywords: ballerina, programming language, edi, contract
 permalink: /learn/edi-tool/
 active: edi-tool
 intro: The EDI tool provides the below set of command line tools to work with EDI files in Ballerina.
----
+--- 
+
+- **Code generation:** Generate Ballerina records and parsing functions for a given EDI schema
+- **Library generation:** Generate Ballerina records, parsing functions, utility methods, and a REST connector for a given collection of EDI schemas, and also, organize those as a Ballerina library
+
+## Compatibility
+
+|                            | Compatible with |
+|----------------------------|------------------|
+| Ballerina language version | 2201.7.0         |
+| Java Development Kit (JDK) | 11               |
```

Comment:
Why do we need to have this ? The tool will be compatible with any future ballerina update versions too and soon it will become JDK 17. So this seems confusing.

Reply:
Removed.